drift/info_parser.py

```diff
@@ -150,46 +151,37 @@ def _get_value_name(name):
         _, _, n = name.partition(".")
         return n
 
-    def _find_group(name, grouped_comps):
-        for group in grouped_comps:
-            if group["name"] == name:
-                return group
-
     def _set_summary_state(grouped_comparison):
-        states = {comparison["state"] for comparison in grouped_comparison["comparisons"]}
+        states = {comparison["state"] for comparison in grouped_comparison}
         if COMPARISON_DIFFERENT in states:
-            grouped_comparison["state"] = COMPARISON_DIFFERENT
+            grouped_comparison[0]["state"] = COMPARISON_DIFFERENT
```

Comment:
What's the magic `0`? :-)

Reply:
Because `get` on the dict, returns a list of values. Here is one sample:
```
[{'name': 'bash', 'state': 'DIFFERENT', 'systems': [{'id': 'ff35596c-f98e-11e9-aea9-98fa9b07d419', 'value': ''}, {'id': '89df6310-f98e-11e9-8a65-98fa9b07d419', 'value': ''}, {'id': 'fc1e497a-28ae-11e9-afd9-c85b761454fa', 'value': '4.4.23-6.fc29.x86_64'}, {'id': 'bbbbbbbb-28ae-11e9-afd9-c85b761454fa', 'value': '4.4.23-6.fc29.x86_64'}, {'id': 'bbbbbbbb-28ae-11e9-afd9-c85b761454fa', 'value': '4.4.23-6.fc29.x86_64'}], 'drifted_from_baseline': False}]
```

it's a list of only 1 result, with dict inside it, that's the reason to get [0]

Reply:
this does not look good @fellipeh. What happens if there is more than one comparison group?

Reply:
fixed